recipe/meta.yaml

```diff
@@ -13,12 +13,13 @@ source:
     - vc14.patch
 
 build:
-  number: 3
+  number: 4
```

Comment:
What's the build number bump for?

Reply:
I thought I had to with the vc update, reverting.

Reply:
Ah ok. Yeah I think Python 3.5 and 3.6 use the exact same compiler and version. So probably not relevant here. Definitely a good thing to be thinking about though. Particularly with Python 3.7 around the corner.

Reply:
Isnt the release in june 2018 ?

Reply:
Yep. Around the corner is relative. 😉